### DISCUSSION TITLE
Documentation for handleRedirectCallback?

### DISCUSSION THREAD

**Discussion by sudowoodo200** [clerk/javascript#3560] [Support request - DO NOT OPEN HERE]
Is there documentation for this anyway? It's only mentioned in passing and not explained

**Comment by wobsoriano** [comment]
https://clerk.com/docs/references/javascript/clerk/handle-navigation#handle-redirect-callback

https://clerk.com/docs/custom-flows/oauth-connections#create-the-sign-up-and-sign-in-flow

**Comment by sudowoodo200** [comment]
Oh yea that helps. But how do we check if the request actually came from Clerk? (For eg. others SDKs may rely on a secret or code that is being sent with the request) Should we just open a GET endpoint, throw all the query params into this function and trust that Clerk will handle it?

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
